reactioncommerce/reaction
# PDP Cleanup - Basic Details Card, Tax Card & Inventory Card

Fixes #2032 

### How to test
- Clone this branch
- Open PDP variant sidebar on an existing/new product
- Open browser console to catch console errors (if any)
- Navigate through the fields/buttons/switches to ensure everything works as it did before with the Blaze template
